src/deployer.ts

```diff
@@ -76,7 +76,7 @@ export abstract class Deployer extends EventEmitter {
   /**
    * Deploy the app.
    */
-  public abstract deploy(): Promise<void>;
+  public abstract deploy(): Promise<void | number>;
```

Comment:
allow for a number to be return.

This could be expanded on to return a more complex type, but I do not see the need given the called deployable reports independently on errors/exceptions it may encounter.